### PR TITLE
Update JUnit

### LINKED ISSUE #254: Apply more JUnit testing for command

### PR DESCRIPTION
**Pull request by hlwang56**
close #254

### CODE REVIEW

File: src/test/java/seedu/planitarium/command/AddPersonTest.java [hunk 1]
```diff
@@ -1,4 +1,59 @@
...
+            "Unknown error is detected from 'Empty string after `/n`', please check again.";
+    protected static final String EMPTY_GROUPINX_ERROR_MSG =
+            "Unknown error is detected from 'Empty string after `/g`', please check again.";
+    protected static final String INVALID_GRPINX_ERROR_MSG =
+            "Unknown error is detected from 'Invalid group index `4`', please check again.";
+
+    @Test
+    void addPerson_invalidName_fail() {
+        try {
+            add = new AddPersonCommand(CommandsForTesting.ADDPERSON4, family);
```
Comment (Teanweijun) [inline src/test/java/seedu/planitarium/command/AddPersonTest.java:24]: Would it be better to include a `fail()` statement right after this command in the `try` block as well? Then you could catch those that did not even throw errors.

File: src/test/java/seedu/planitarium/command/AddRecordTest.java [hunk 2]
```diff
@@ -1,4 +1,129 @@
...
+        family = new Family();
+        AddPersonCommand add = new AddPersonCommand(CommandsForTesting.ADDPERSON, family);
+        add.execute();
+    }
+
+    @Test
+    void addRec_invalidUid_fail() {
+        try {
+            initialize();
+            addRec = new AddRecordCommand(CommandsForTesting.ADDINCOME2, family);
```
Comment (Teanweijun) [inline src/test/java/seedu/planitarium/command/AddRecordTest.java:43]: Same comment as that in `AddPersonTest.java`

File: src/test/java/seedu/planitarium/command/AddRecordTest.java [hunk 3]
```diff
@@ -1,4 +1,129 @@
...
+            addOut = new AddRecordCommand(CommandsForTesting.ADDEXPEND3, family);
+        } catch (PlanITariumException e) {
+            assertEquals(e.toString(), INVALID_CAT_MSG);
+        } catch (Exception e) {
+            fail();
+        }
+    }
+
+    @Test
+    void addRec_success() {
```
Comment (Teanweijun) [inline src/test/java/seedu/planitarium/command/AddRecordTest.java:118]: Did you forget to specify the type of inputs that `AddRecordCommand` will be given?

File: src/test/java/seedu/planitarium/command/DeletePersonTest.java [hunk 4]
```diff
@@ -1,4 +1,60 @@
...
+        family = new Family();
+        AddPersonCommand add = new AddPersonCommand(CommandsForTesting.ADDPERSON, family);
+        add.execute();
+    }
+
+    @Test
+    void deletePerson_invalidUid_fail() {
+        try {
+            initialize();
+            delete = new DeletePersonCommand(CommandsForTesting.DELETEPERSON2, family);
```
Comment (Teanweijun) [inline src/test/java/seedu/planitarium/command/DeletePersonTest.java:31]: Same comment as that in `AddPersonTest.java`

File: src/test/java/seedu/planitarium/command/DeleteRecordTest.java [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+        AddRecordCommand addOut = new AddRecordCommand(CommandsForTesting.ADDINCOME, family);
+        addIn.execute();
+        addOut.execute();
+    }
+
+    @Test
+    void deleteRec_invalidUid_fail() {
+        try {
+            initialize();
+            deleteIn = new DeleteRecordCommand(CommandsForTesting.DELETEINCOME4, family);
```
Comment (Teanweijun) [inline src/test/java/seedu/planitarium/command/DeleteRecordTest.java:41]: Same comment as that in AddPersonTest.java

File: src/test/java/seedu/planitarium/command/EditRecordTest.java [hunk 6]
```diff
@@ -1,4 +1,135 @@
...
+        AddRecordCommand addOut = new AddRecordCommand(CommandsForTesting.ADDINCOME, family);
+        addIn.execute();
+        addOut.execute();
+    }
+
+    @Test
+    void deleteRec_invalidUid_fail() {
+        try {
+            initialize();
+            editIn = new EditRecordCommand(CommandsForTesting.EDITINCOME6, family);
```
Comment (Teanweijun) [inline src/test/java/seedu/planitarium/command/EditRecordTest.java:49]: Same comment as that in AddPersonTest.java

File: src/test/java/seedu/planitarium/command/ListByGroupTest.java [hunk 7]
```diff
@@ -1,4 +1,51 @@
...
+        AddRecordCommand addOut = new AddRecordCommand(CommandsForTesting.ADDINCOME, family);
+        addIn.execute();
+        addOut.execute();
+    }
+
+    @Test
+    void listByGroup_invalidGroupInx_fail() {
+        try {
+            initialize();
+            list = new ListCommand(CommandsForTesting.LISTBYGROUP2, family);
```
Comment (Teanweijun) [inline src/test/java/seedu/planitarium/command/ListByGroupTest.java:34]: Same comment as that in AddPersonTest.java

File: src/test/java/seedu/planitarium/command/SearchTest.java [hunk 8]
```diff
@@ -1,4 +1,67 @@
...
+        AddRecordCommand addOut = new AddRecordCommand(CommandsForTesting.ADDINCOME, family);
+        addIn.execute();
+        addOut.execute();
+    }
+
+    @Test
+    void find_invalidDes_fail() {
+        try {
+            initialize();
+            search = new SearchCommand(CommandsForTesting.FIND2, family);
```
Comment (Teanweijun) [inline src/test/java/seedu/planitarium/command/SearchTest.java:37]: Same comment as that in AddPersonTest.java

**Review by Teanweijun — COMMENTED** [review]
General structure looks fine. I've left some comments for you to look at that you might want to consider.

**Review by 1szheng — APPROVED** [review]
Looks good, comments addressed

**Pull request merged**